tests/slow_tests/test_benchmarks.py

```diff
@@ -1,37 +1,101 @@
 import unittest
 import os
+import json
+import shutil
 
-from flow.benchmarks.figureeight0 import flow_params as figureeight0
-from flow.benchmarks.figureeight1 import flow_params as figureeight1
-from flow.benchmarks.figureeight2 import flow_params as figureeight2
-from flow.benchmarks.merge0 import flow_params as merge0
-from flow.benchmarks.merge1 import flow_params as merge1
-from flow.benchmarks.merge2 import flow_params as merge2
+import ray
+import ray.rllib.agents.ppo as ppo
 
-from flow.benchmarks.baselines.bottleneck0 import bottleneck0_baseline
-from flow.benchmarks.baselines.bottleneck1 import bottleneck1_baseline
-from flow.benchmarks.baselines.bottleneck2 import bottleneck2_baseline
-from flow.benchmarks.baselines.figureeight012 import figure_eight_baseline
-from flow.benchmarks.baselines.grid0 import grid0_baseline
-from flow.benchmarks.baselines.grid1 import grid1_baseline
-from flow.benchmarks.baselines.merge012 import merge_baseline
+from ray.rllib.agents.agent import get_agent_class
+from ray.tune.registry import register_env
 
-os.environ["TEST_FLAG"] = "True"
+from flow.utils.registry import make_create_env
+from flow.utils.rllib import FlowParamsEncoder
 
 
-class TestBaselines(unittest.TestCase):
+import flow.benchmarks.bottleneck0 as bottleneck0
+import flow.benchmarks.bottleneck1 as bottleneck1
+import flow.benchmarks.bottleneck2 as bottleneck2
+import flow.benchmarks.figureeight0 as figureeight0
+import flow.benchmarks.figureeight1 as figureeight1
+import flow.benchmarks.figureeight2 as figureeight2
+import flow.benchmarks.grid0 as grid0
+import flow.benchmarks.grid1 as grid1
+import flow.benchmarks.merge0 as merge0
+import flow.benchmarks.merge1 as merge1
+import flow.benchmarks.merge2 as merge2
+
+N_CPUS = 1
+ray.init(num_cpus=N_CPUS, redirect_output=True)
+
+os.environ['TEST_FLAG'] = 'True'
+
+
+class TestBenchmarks(unittest.TestCase):
+
     """
     Tests that the baselines in the benchmarks folder are running and
     returning expected values (i.e. values that match those in the CoRL paper
     reported on the website, or other).
     """
 
+    """
+    self.env, self.scenario = setup_bottlenecks()
+        self.exp = SumoExperiment(self.env, self.scenario)
+        self.exp.run(5, 50)
+    """
+
+    def setup(self):
+        if not os.path.exists('./benchmark_tmp'):
+            os.mkdir('benchmark_tmp')
+
+    def tearDown(self):
+        shutil.rmtree('benchmark_tmp')
+
+    def ray_runner(self, num_runs, flow_params, version):
+        alg_run = 'PPO'
+        HORIZON = 10
+
+        agent_cls = get_agent_class(alg_run)
+        config = agent_cls._default_config.copy()
+        config['num_workers'] = 1
+        config['train_batch_size'] = 200  # arbitrary
```

Comment:
can you try decreasing train_batch_size to 50 or smaller and see if that gives a little speed_boost?